docs/source/challenge_creation.md

```diff
@@ -2,9 +2,9 @@
 
 One can create a challenge in EvalAI using following methods:
 
-* Challenge creation using zip configuration
+* Using zip configuration
```

Comment:
I think the earlier version was more correct. What you can add here is `zip file configuration` instead of `zip configuration`. Please revert it.